JavaScript-3 Assignments/Question3/js/script.js

```diff
@@ -0,0 +1,48 @@
+var webWorker = {}; // Global variable declaration.
+
+/**
+ * Checks if web worker is compatible with the browser.
+ * If the webWorker is of object type it calls the web_worker.js file.
+ */
+function startWorker() {
+    var result = document.getElementById('result');
+    if ('undefined' !== typeof (Worker)) {
+        if ('object' === typeof (webWorker)) {
+            webWorker = new Worker('js/web_worker.js');
```

Comment:
It always creates a new instance of webworker, without considering last webworker.